libs/sdk-ui-filters/src/RankingFilter/MeasureDropdown/MeasureDropdownBody.tsx

```diff
@@ -0,0 +1,43 @@
+// (C) 2020 GoodData Corporation
+import React from "react";
+import Overlay from "@gooddata/goodstrap/lib/core/Overlay";
+import { Identifier } from "@gooddata/sdk-model";
+import { MeasureDropdownItem } from "./MeasureDropdownItem";
+import { IMeasureDropdownItem } from "../types";
+
+interface IMeasureDropdownBodyProps {
+    items: IMeasureDropdownItem[];
+    selectedItemIdentifier: Identifier;
```

Comment:
I would name it `selectedItemLocalIdentifier`. Imho `Identifier` is something else. See attribute class as well.